### PR TITLE
feat: Added endpoints for create, delete, change geometry for road segment outlines + change attributes endpoint

### PR DESCRIPTION
**Pull request by janlesage**
- WR-596
- WR-597
- WR-598

### CODE REVIEW

File: src/Public.Api/RoadSegment/RoadSegmentController-ChangeAttributes.cs [hunk 1]
```diff
@@ -0,0 +1,86 @@
+namespace Public.Api.RoadSegment;
```
Comment (ArneD) [inline src/Public.Api/RoadSegment/RoadSegmentController-ChangeAttributes.cs:1]: blocked namespace

File: src/Public.Api/RoadSegment/RoadSegmentController-ChangeOutlineGeometry.cs [hunk 2]
```diff
@@ -0,0 +1,90 @@
+namespace Public.Api.RoadSegment;
```
Comment (ArneD) [inline src/Public.Api/RoadSegment/RoadSegmentController-ChangeOutlineGeometry.cs:1]: blocked namespace

File: src/Public.Api/RoadSegment/RoadSegmentController-ChangeAttributes.cs [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    [SwaggerResponseHeader(StatusCodes.Status202Accepted, "ETag", "string", "De ETag van de response.")]
+    [SwaggerResponseHeader(StatusCodes.Status202Accepted, "x-correlation-id", "string", "Correlatie identificator van de response.")]
+    [SwaggerResponseExample(StatusCodes.Status400BadRequest, typeof(BadRequestResponseExamples))]
+    [SwaggerResponseExample(StatusCodes.Status404NotFound, typeof(RoadSegmentNotFoundResponseExamples))]
+    [SwaggerResponseExample(StatusCodes.Status412PreconditionFailed, typeof(PreconditionFailedResponseExamples))]
+    [SwaggerResponseExample(StatusCodes.Status429TooManyRequests, typeof(TooManyRequestsResponseExamples))]
+    [SwaggerResponseExample(StatusCodes.Status500InternalServerError, typeof(InternalServerErrorResponseExamples))]
+    [SwaggerRequestExample(typeof(ChangeRoadSegmentAttributesParameters), typeof(ChangeRoadSegmentAttributesParametersExamples))]
+    [SwaggerOperation(OperationId = nameof(ChangeAttributes), Description = "Attributen wijzigen van een wegsegment: status, toegangsbeperking, wegklasse, wegbeheerder en wegcategorie.")]
+    public async Task<IActionResult> ChangeAttributes(
```
Comment (ArneD) [inline src/Public.Api/RoadSegment/RoadSegmentController-ChangeAttributes.cs:53]: Name should match toggle without `Toggle` to work correctly by convention.
So this should be `RoadSegmentChangeAttributes`.

Comment (ArneD) [reply src/Public.Api/RoadSegment/RoadSegmentController-ChangeAttributes.cs:53]: and check other actions as well

File: src/Public.Api/RoadSegment/RoadSegmentController-ChangeOutlineGeometry.cs (hunk 2 above)
Comment (ArneD) [reply src/Public.Api/RoadSegment/RoadSegmentController-ChangeOutlineGeometry.cs:1]: for other classes as well

**Review by ArneD — CHANGES_REQUESTED** [review]

File: src/Public.Api/RoadSegment/RoadSegmentController-ChangeAttributes.cs (hunk 3 above)
Comment (janlesage) [reply src/Public.Api/RoadSegment/RoadSegmentController-ChangeAttributes.cs]: The class names for FeatureToggles are with Toggle in every other part of the API also? In configuration this needs to be without Toggle indeed, which it is. Are we missing something?

Comment (janlesage) [reply src/Public.Api/RoadSegment/RoadSegmentController-ChangeAttributes.cs]: Normalized actions and toggles and route definitions

**Review by ArneD — APPROVED** [review]

**Pull request merged**